### PR TITLE
[doc] replace ` jaeger exporter` with `otlp http exporter`

### PR DESCRIPTION
**Pull request by Petrie**
Signed-off-by: Peter Liu <lpfvip2008@gmail.com>

Resolve https://github.com/open-telemetry/opentelemetry.io/issues/1567

### CODE REVIEW

File: website_docs/exporting_data.md
Comment (dmathieu) [inline website_docs/exporting_data.md:68]: Since I think only the most recent versions of jaeger support OTLP, should we mention the minimum required version?

Comment (Petrie) [reply website_docs/exporting_data.md]: Added. Jaeger support OTLP after release [v1.35.0](https://github.com/jaegertracing/jaeger/releases/tag/v1.35.0)
Thanks for reminding

**Comment by dmathieu** [comment]
Sorry I didn't see it before, but this PR will conflict with #3705. Since that other PR is touching the same files, how about including this change there?

**Comment by svrnm** [comment]
> Sorry I didn't see it before, but this PR will conflict with #3585.

@Petrie happy to work with you getting 3585 and your PR here merged together.

**Comment by Petrie** [comment]
> > Sorry I didn't see it before, but this PR will conflict with #3585.
> 
> @Petrie happy to work with you getting 3585 and your PR here merged together.

I understand a conflict will occur when pr https://github.com/open-telemetry/opentelemetry-go/pull/3585 get merged.
But i am a little confused , anything else should I do to avoid the confilct?

**Comment by svrnm** [comment]
> > > Sorry I didn't see it before, but this PR will conflict with #3585.
> > 
> > 
> > @Petrie happy to work with you getting 3585 and your PR here merged together.
> 
> I understand a conflict will occur when pr #3585 get merged. But i am a little confused , anything else should I do to avoid the confilct? @damemi @svrnm

I think there's 2 ways to go about that:

* one PR gets merged first and the other one needs to fix the conflict
* one PR gets incorporated into the other one first (via a PR to the forked branch)

I'll leave it to @dmathieu to guide us here :-) I'm ok if we merge them sequentially and one of us needs to do some extra work (can be me)

**Comment by dmathieu** [comment]
I don't have a preference either. I guess it really depends on when a maintainer does the merging.

**Comment by Petrie** [comment]
ok, I get it. I'll wait.
I think it's a simple conflict. if it is possible, I'll take care of it.

**Comment by svrnm** [comment]
> Ok, I get it. I'll wait. I think it's a simple conflict. if it is possible, I'll take care of it.

thank you!

**Review by svrnm — APPROVED** [review]

File: website_docs/exporters.md
Comment (pellared) [inline website_docs/exporters.md:55]:
```suggestion
docker run -d --name jaeger \
  -e COLLECTOR_OTLP_ENABLED=true \
  -p 16686:16686 \
  -p 4318:4318 \
  jaegertracing/all-in-one:latest
```

I think only this is needed

**Review by pellared — APPROVED** [review]

**Review by MadVikingGod — APPROVED** [review]

**Pull request merged**
